### PR TITLE
Add docs for token revocation via session termination api

### PR DESCRIPTION
**Pull request by piraveena**
A part of https://github.com/wso2/product-is/issues/10843

### CODE REVIEW

**Review by ayshsandu — APPROVED** [review]

File: en/docs/develop/session-mgt-rest-api.md [hunk 1]
```diff
@@ -19,6 +19,16 @@ template: templates/swagger.html
        4. Add a `-k` header to the curl command and run the curl command on the terminal with a running instance of WSO2
          IS. 
          
+!!! Note
+    
+    - From IS5.12 version onwards, We revoke the access tokens  mapped with the session via REST API.
+    - This is applicable for authorization code grant, implicit flow, refresh token grant and hybrid flow.
```
Comment (sherenewso2) [inline en/docs/develop/session-mgt-rest-api.md:25]:
```suggestion
    - From WSO2 IS 5.12.0 onwards, access tokens that are mapped with the session via REST API can be revoked. This is applicable for authorization code grant, implicit flow, refresh token grant, and hybrid flow. 
```

File: en/docs/develop/session-mgt-rest-api.md [hunk 2]
```diff
@@ -19,6 +19,16 @@ template: templates/swagger.html
        4. Add a `-k` header to the curl command and run the curl command on the terminal with a running instance of WSO2
          IS. 
          
+!!! Note
+    
+    - From IS5.12 version onwards, We revoke the access tokens  mapped with the session via REST API.
+    - This is applicable for authorization code grant, implicit flow, refresh token grant and hybrid flow. 
+    - If the same accesstoken is used my multiple sessions, then if one session is terminated via API, 
+      the other sessions may loose the tokens. So it is recommended to use sso-session binding if you want to have the 
+      functionality to revoke the accesstoken when the session is termianted via REST API.
```
Comment (sherenewso2) [inline en/docs/develop/session-mgt-rest-api.md:28]:
```suggestion
    -  If one session is terminated via the API and the same access token is used by multiple sessions,
      the other sessions may then lose their tokens. Therefore, it is recommended to use sso-session binding if you wish to retain the capability to revoke the access token when the session is terminated via REST API.
```

File: en/docs/develop/session-mgt-rest-api.md [hunk 3]
```diff
@@ -19,6 +19,16 @@ template: templates/swagger.html
...
+    - WSO2 Identity Server sends SAML and OIDC back-channel logouts also when session is termianted via API. To support 
+      this, the application should be configured back-channel logouts.
```
Comment (sherenewso2) [inline en/docs/develop/session-mgt-rest-api.md:30]:
```suggestion
    - WSO2 Identity Server sends SAML and OIDC back-channel logouts also when session is terminated via API. To support 
      this, the application should be configured to enable back-channel logout.
```

Comment (sherenewso2) [reply en/docs/develop/session-mgt-rest-api.md:30]: "sends SAML and OIDC back-channel logouts" --> it is not very clear what is meant by this. What are we trying to say here?

File: en/docs/learn/configuring-openid-connect-back-channel-logout.md [hunk 4]
```diff
@@ -219,3 +219,7 @@ the newly registered service provider:
     `           null          ` indicating that you are also logged out
     from the playground2 application.  
     ![]( ../assets/img/112390325/112391429.png) 
+
+!!! Note
+    When a session is terminated via the [Session Management REST API](../develop/session-mgt-rest-api)], 
+    WSO2 Identity Server will send back-channel logout notification.
```
Comment (sherenewso2) [inline en/docs/learn/configuring-openid-connect-back-channel-logout.md:225]:
```suggestion
    WSO2 Identity Server will send a back-channel logout notification.
```

File: en/docs/learn/oauth2-token-revocation.md [hunk 5]
```diff
@@ -86,3 +86,9 @@ curl -X POST --basic -u "<client id>:<client secret>" -H "Content-Type: applicat
 ```
 
 The `         callback        ` parameter is optional.
+
+!!! Note
+    When a session is terminated via rest api, WSO2 Identity Server will 
+    revoke the mapped accesstoken as well. There can be some instances where same access token is used across
```
Comment (sherenewso2) [inline en/docs/learn/oauth2-token-revocation.md:92]:
```suggestion
    revoke the mapped access token as well. There can be some instances where the same access token is used across 
```

File: en/docs/learn/oauth2-token-revocation.md [hunk 6]
```diff
@@ -86,3 +86,9 @@ curl -X POST --basic -u "<client id>:<client secret>" -H "Content-Type: applicat
 ```
 
 The `         callback        ` parameter is optional.
+
+!!! Note
+    When a session is terminated via rest api, WSO2 Identity Server will 
+    revoke the mapped accesstoken as well. There can be some instances where same access token is used across 
+    multiple sessions. So it is always recommended to use sso-session binding if you are using session management
```
Comment (sherenewso2) [inline en/docs/learn/oauth2-token-revocation.md:93]:
```suggestion
    multiple sessions. Therefore, it is always recommended to use sso-session binding if you are using the session management 
```

**Review by sherenewso2 — APPROVED** [review]

**Pull request merged**